toolchain/BUILD

```diff
@@ -0,0 +1,90 @@
+load(":cc_toolchain_config.bzl", "cc_toolchain_config")
+package(default_visibility = ["//visibility:public"])
+
+filegroup(name = "empty")
+
+
+cc_toolchain_suite(
+    name = "clang_suite",
+    toolchains = {
+        "aarch64": ":cc-compiler-aarch64",
+        "ppcle": "cc-compiler-ppcle",
+        "s390x": ":cc-compiler-s390x",
+        "x86_32": ":cc-compiler-x86_32",
+        "x86_64": ":cc-compiler-x86_64",
+    },
+)
+
+CC_TOOLCHAIN_CPUS = [
```

Comment:
Would it be possible to condense the toolchains dictionary on L9 and this into a single variable that can be reused in the two places? That way you'd be able to use cpu + name from the dictionary directly in the for loop of cc_toolchain, and you'd have less duplication here.

Reply:
Done!